plugin/trino-clickhouse/src/test/resources/config.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0"?>
+<yandex>
+    <keep_alive_timeout>10</keep_alive_timeout>
```

Comment:
Why? add code comment here & in cmt message too.

Reply:
The purpose is fixing flaky server as https://github.com/ClickHouse/clickhouse-jdbc/issues/717.